pkg/policies/binary/binary.go

```diff
@@ -107,7 +107,10 @@ func (l *logger) Debug(desc string, args ...interface{}) {
 func (b Binary) Check(ctx context.Context, c *github.Client, owner,
 	repo string) (*policydef.Result, error) {
 	oc, rc := getConfig(ctx, c, owner, repo)
-	enabled := config.IsEnabled(oc.OptConfig, rc.OptConfig, repo)
+	enabled, err := config.IsEnabled(ctx, oc.OptConfig, rc.OptConfig, c.Repositories, owner, repo)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Looks like `config.IsEnabled` will need to be mocked now for all policies, similar to how `configFetchConfig` is currently mocked. It probably should have been mocked to begin with, as the policy tests don't need to unit test `IsEnabled`. The policy tests should now have the return value of `IsEnabled` be an input instead of the org/repo optconfigs.

Reply:
Thanks. Updated with c59738bb78b1e2b816fed5dc7ab8f96feaedf30d.